src/utils/sleep.ts

```diff
@@ -1,10 +1,8 @@
-export default {
-  /**
-   * Method that "sleeps" for specified timeout. Must be called from async method.
-   * @param ms Delay to sleep, in milliseconds.
-   * @returns A promise to await upon.
-   */
-  sleep (ms: number): Promise<void> {
-    return new Promise(resolve => setTimeout(resolve, ms))
-  }
+/**
+ * Function that "sleeps" for specified timeout. Must be called from async method.
+ * @param ms Delay to sleep, in milliseconds.
+ * @returns A promise to await upon.
+ */
+export function sleep (ms: number): Promise<void> {
+  return new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
Just curious here, as far as i can tell this is just used in jest spec files. Did you not solve a similar issue with `flushPromises()` or was something unrelated?

Reply:
Yes, and I do use it in the unit test to wait for the debounce interval to pass.

Other than this, this was a pretty handy function for prototype actions (like save), as we did in the Business Create UI, remember?

Reply:
I do! It is very handy indeed, i'm not opposed to it whatsoever 👍